index.html

```diff
@@ -7680,6 +7681,120 @@ <h5>Presentational Roles Conflict Resolution</h5>
+						A valid month string: <code><span role="time">2019-11</span></code>
+					</li>
+					<li>
+						A valid date string: <code><span role="time">2019-11-18</span></code>
+					</li>
+					<li>
+						A valid yearless date string: <code><span role="time">11-18</span></code>
+					</li>
+					<li>
+						A valid time string: <code><span role="time">09:54:39</span></code>
+					</li>
+					<li>
+						A valid floating date and time string: <code><span role="time">2019-11-18T14:54</span></code>
+					</li>
+					<li>
+						A valid time-zone offset string: <code><span role="time">-08:00</span></code>
+					</li>
+					<li>
+						A valid global date and time string: <code><span role="time">2019-11-18T14:54Z</span></code>
+					</li>
+					<li>
+						A valid week string: <code><span role="time">2019-W47</span></code>
+					</li>
+					<li>
+						Four or more ASCII digits, at least one of which is not U+0030 DIGIT ZERO (0): <code><span role="time">0001</span></code>
+					</li>
+					<li>
+						A valid duration string: <code><span role="time">4h 18m 3s</span></code>
+					</li>
+				</ul>
+			</div>
+			<table class="role-features">
+				<caption>Characteristics:</caption>
+				<thead>
+					<tr>
+						<th scope="col">Characteristic</th>
+						<th scope="col">Value</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<th class="role-abstract-head" scope="row">Is Abstract:</th>
+						<td class="role-abstract"> </td>
+					</tr>
+					<tr>
+						<th class="role-parent-head" scope="row">Superclass Role:</th>
+						<td class="role-parent"><rref>section</rref></td>
+					</tr>
+					<tr>
+						<th class="role-children-head" scope="row">Subclass Roles:</th>
+						<td class="role-children"> </td>
+					</tr>
+					<tr>
+						<th class="role-base-head" scope="row">Base Concept:</th>
+						<td class="role-base"> </td>
+					</tr>
+					<tr>
+						<th class="role-related-head" scope="row">Related Concepts:</th>
+						<td class="role-related"><a href="https://www.w3.org/TR/html52/textlevel-semantics.html#the-time-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>time</code></a></td>
+					</tr>
+					<tr>
+						<th class="role-scope-head" scope="row">Required Context Role:</th>
+						<td class="role-scope"> </td>
+					</tr>
+					<tr>
+						<th class="role-mustcontain-head" scope="row">Required Owned Elements:</th>
+						<td class="role-mustcontain"> </td>
+					</tr>
+					<tr>
+						<th class="role-required-properties-head">Required States and Properties:</th>
+						<td class="role-required-properties"> </td>
+					</tr>
+					<tr>
+						<th class="role-properties-head" scope="row">Supported States and Properties:</th>
+						<td class="role-properties"> </td>
+					</tr>
+					<tr>
+						<th class="role-inherited-head" scope="row">Inherited States and Properties:</th>
+						<td class="role-inherited"> </td>
+					</tr>
+					<tr>
+						<th class="role-namefrom-head" scope="row">Name From:</th>
+						<ul>
```

Comment:
When It is a single value we don't normally use a list in Name From. I wonder if this will fix it.

```suggestion
	<td class="role-namefrom">author</td>
```

Reply:
(couldn't work out how to suggest a change for multiple lines - sorry but this should replace the next 2 too!)

Reply:
@jnurthen weirdness is fixed! https://pr-preview.s3.amazonaws.com/melanierichards/aria/pull/895.html#time

I originally had more than one item in the list and just didn't simplify the markup when I made the change. But I think the missing `<td>` was more of the problem. Not sure how I missed that and was on the point of just running this through a validator, but in any case, fixed now. Thank you!